### ISSUE TITLE
HOST described in .env is not applied

### ISSUE THREAD

**Issue by maguro869** [denverquane/automuteus#217]
.env
```
BRANCH=3.1.0

DISCORD_BOT_TOKEN=<TOKEN>
HOST=http://example.com:8123
```
run 
```
sudo docker-compose pull
sudo docker-compose up -d
``` 
and run ".au n" in Discord,
bot send DM this Embed message
```
You just started a game!
Click the following link to link your capture:
<Invalid HOST provided (should resemble something like http://localhost:8123)>

Don't have the capture installed?

To link your capture manually:
URL
Invalid HOST provided
Code.
0B6A492E
```
 and the originally entered HOST is not applied.
Is there any way to solve this problem?
![rapture_20201104174538](https://user-images.githubusercontent.com/50356023/98090713-c1949f00-1ec7-11eb-9ae9-e0bdd1e91e07.png)

Translated with DeepL sry xd

**Comment by vmmelo** [comment]
are you running it on your local machine, I think you only need to set 
`HOST=http://localhost:8123`

**Comment by maguro869** [comment]
running cloud server(ConoHa VPS).

I want to display something like "aucaputure://example.com:8123/0B6A492E" in the Embed place, but it doesn't show up when I edit the environment variable ;w;

I followed the steps in the Self-Hosting Docker Compose: in README.md to configure it, but it didn't show up like the official Bot.

**Comment by ShawnHardwick** [comment]
Are you still experiencing this issue on 6.0.2?

**Comment by denverquane** [comment]
Closing due to inactivity; feel free to reopen if you still have this issue

**Comment by ama1r0** [comment]
I tried to do self-hosting with Docker, but I got the symptoms shown in the image above.
When I checked Docker, I found that "filename_galactus_1" could not be started properly.
The content of the .env is
GALACTUS_HOST =http://localhost:8123
GALACTUS_EXTERNAL_PORT=8123
Error message
for galactus Cannot start service galactus: driver failed programming external connectivity on endpoint "filename_galactus_1" (*****): Bind for 0.0.0. 0:8123 failed: port is already allocated

Translated with www.DeepL.com/Translator (free version)
